Lesson_5/Task_1.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Task_1</title>
+    <style>
+        td {
+            border: 1px solid black;
+            height: 40px;
+            text-indent: 20px;
+        }
+        th {
+            border: 1px solid black;
+            height: 40px;
+        }
+    </style>
+</head>
+<body>
+    <div style="margin: 20px 100px">
+<!--        Я обернул всё в div с маржином, потому что таблица от края до края выглядит по-уродски :)-->
+        <h2 style="text-align: center">Добыча драгоценных камней</h2>
+        <table style="border: 1px solid black;border-collapse: collapse; width: 100%; table-layout: fixed; min-width: 400px;">
+            <tr style="background-color: #008100;">
+                <th style="width: 100px"></th>
+                <th>2010</th>
+                <th>2011</th>
+                <th>2012</th>
+            </tr>
+            <tr style="background-color: #ffffcf;">
+                <td style="width: 100px">Сапфиры</td>
```

Comment:
I think, you can set width: 100px only for first column

Reply:
done